operations/experimental/add-metric-store.yml

```diff
@@ -1,169 +0,0 @@
-- type: replace
-  path: /addons/name=prom_scraper/exclude?/instance_groups/-
-  value: metric-store
```

Comment:
Looks like this operation of excluding the prom_scrapper from the metrics-store instance group was not replaced inside of the base manifest. Is this the correct behavior? Does the metrics-store now need to have the prom_scraper addon?